source/routing/redirection.md

```diff
@@ -77,17 +83,22 @@ App.TopChartsChooseRoute = Ember.Route.extend({
 });
 
 // Superclass to be used by all of the filter routes below
-App.FilterRoute = Ember.Route.extend({
+// app/filter/route.js
+export default Ember.Route.extend({
   activate: function() {
     var controller = this.controllerFor('application');
     controller.set('lastFilter', this.templateName);
   }
 });
 
-App.TopChartsSongsRoute = App.FilterRoute.extend();
-App.TopChartsAlbumsRoute = App.FilterRoute.extend();
-App.TopChartsArtistsRoute = App.FilterRoute.extend();
-App.TopChartsPlaylistsRoute = App.FilterRoute.extend();
+// app/top-charts-songs/route.js
```

Comment:
O_O.

I'm reauditing the routes section. There appear to be many find/replace errors and places where find/replace didn't properly find at all. Will republish 1.11